Libraries/Components/FocusableWindows/FocusableWindows.windows.js

```diff
@@ -86,7 +86,8 @@ class FocusableWindowsTemplate {
       'auto',
       'yes',
       'no',
-      'no-hide-descendants'
+      'no-hide-descendants',
+      'yes-dont-hide-descendants' // Windows only, not in ReactNative
```

Comment:
>, not in ReactNative [](start = 49, length = 20)

Remove this "not in ReactNative" thing, it's ambiguous.
Also, where is importantForAccessibility defined for normal views? Perhaps in the RN ViewPropTypes.js. In order to modify that you would have to bring the file as a copy.

This public RNW does have support for adding properties to View WITHOUT bringing a copy in (check the PlatformViewPropTypes.windows.js file), but I don't think it allows "redefinition". But maybe you're lucky and it does (with a "last one wins" kind of way).